src/style.css

```diff
@@ -0,0 +1,122 @@
+html {
+  margin: 0;
+  display: flex;
```

Comment:
do not change  `display` for the whole `html`

if you need, just create separate blocks